packages/core/test/lib/integrations/inboundfilters.test.ts

```diff
@@ -177,6 +177,18 @@ const SENTRY_EVENT: Event = {
   },
 };
 
+const BOGUS_EVENT: Event = {
+  message: '',
+  exception: {
+    values: [
+      {
+        type: 'Error',
+        value: 'undefined',
```

Comment:
I am not sure if this is a scary change, in JS you never know.
But how useful can an undefined error be?

Reason I added it is because, apparently, not all `ResizeObserver` has a value.

Reply:
Good call - I leave `undefined` away for now.